README.md

```diff
@@ -20,31 +20,31 @@ Here we've collected examples of library use cases:
 
 ### 1. Capturing credit card data
 
-[Code example is here](examples/v2/credit-card-example)
+[Code example is here](examples/1-usecases/credit-card-example)
```

Comment:
what is the reason for `1-` prefix? can we simplify?

Reply:
The reason is to sort examples from the most general to the most specific, similar to documentation flow.
![image](https://user-images.githubusercontent.com/29078278/115745691-561e8b00-a39c-11eb-8b36-90aa94f89dc4.png)

Reply:
I would remove numbers they're a bit confusing also what if decide to change general / specific things or add smth in between. Can we remove numbers?

Reply:
Agree - we need to remove digits